services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -52,6 +48,16 @@ export default class IndexPage extends React.Component<Props, State> {
               email: '',
               confirmEmail: '',
             }}
+            validate={values => {
+              // same as above, but feel free to move this into a class method now.
+              let errors = {};
+              if (this.email.value != this.confirmEmail.value) {
```

Comment:
hang on. Don't add new stuff to this PR right now. Do the confirm email on a different branch.

Reply:
Yeah that was by accident. I will correct that now